analyzer/src/main/kotlin/managers/utils/MavenSupport.kt

```diff
@@ -107,20 +109,17 @@ private val File?.safePath: String
 
 class MavenSupport(private val workspaceReader: WorkspaceReader) {
     companion object : Logging {
-        private const val MAX_DISK_CACHE_SIZE_IN_BYTES = 1024L * 1024L * 1024L
-        private const val MAX_DISK_CACHE_ENTRY_AGE_SECONDS = 6 * 60 * 60
-
         // See http://maven.apache.org/pom.html#SCM.
         private val SCM_REGEX = Pattern.compile("scm:(?<type>[^:@]+):(?<url>.+)")!!
         private val USER_HOST_REGEX = Pattern.compile("scm:(?<user>[^:@]+)@(?<host>[^:]+):(?<url>.+)")!!
 
         private val WHITESPACE_REGEX = Regex("\\s")
 
-        private val remoteArtifactCache =
-            DiskCache(
-                ortDataDirectory.resolve("cache/remote_artifacts"),
-                MAX_DISK_CACHE_SIZE_IN_BYTES, MAX_DISK_CACHE_ENTRY_AGE_SECONDS
-            )
+        private val remoteArtifactCache = DiskCache(
+            directory = ortDataDirectory.resolve("cache/remote-artifacts"),
+            maxCacheSizeInBytes = 1024L * 1024L * 1024L,
+            maxCacheEntryAgeInSeconds = 6.hours.inWholeSeconds
```

Comment:
BTW, I don't think that these historic number make sense still. IMO we should use a smaller cache with a larger max cache age.

Can anyone share their current cache sizes on disk?

Reply:
@mnonnenmacher and @fviernau, my "cache/remote_artifacts" is just several hundred KB, what about yours?